benchexec/tools/avr.py

```diff
@@ -15,13 +15,21 @@ class Tool(benchexec.tools.template.BaseTool2):
     URL: https://github.com/aman-goel/avr
     """
 
+    REQUIRED_PATHS = ["build/"]
+
     def executable(self, tool_locator):
         return tool_locator.find_executable("avr.py")
 
     def name(self):
         return "AVR"
 
     def cmdline(self, executable, options, task, rlimits):
+        from math import ceil
```

Comment:
I don't see a reason why this needs to be a local import, so please make it a standard module-level import.

Reply:
Done in a37525b.